docs/guides/e2e-test-tips.md

```diff
@@ -0,0 +1,73 @@
+# E2E tests tips
+
+E2E tests are less stable and slower than unit tests therefore it's useful to know some tricks
+
+## Faster environment setup
+
+When e2e tests are executed here are the steps
+
+1) Build containers
+2) Start Kubernetes Cluster 2
```

Comment:
Are they both Cluster 2, or should this be Cluster 1 ?

Reply:
should be cluster1